concepts/80 Troubleshooting/50 A Drop-Down Editor Does Not Show Data/10 DataField and ValueExpr Should Store The Same Keys.md

```diff
@@ -0,0 +1,98 @@
+To correctly substitute displayed values with required values in a DataGrid/TreeList lookup column, follow these steps:
```

Comment:
This part only marginally differs from the part that follows... We say that two fields must store the same values. Then we also emphasize that they should be of the same type. If they store the same values, then obviously the type is the same, right? 

Also - this section doesn't look like a part of a troubleshooting guide. It has more in common with a Getting Started. Shows me how to set up a lookup from scratch. 

Long story short - I think we can skip this entire step.